FreeRTOS-Plus/Source/FreeRTOS-IoT-Libraries-LTS-Beta2/c_sdk/platform/freertos/transport/include/plaintext_freertos.h

```diff
@@ -47,6 +47,9 @@
 
 /************ End of logging configuration ****************/
 
+/* FreeRTOS+TCP include. */
+#include "FreeRTOS_Sockets.h"
```

Comment:
With this change we could delete it from here too:
https://github.com/FreeRTOS/FreeRTOS/blob/5e76ff435bb7d4625dbc3959d94c8f25e0996be2/FreeRTOS-Plus/Demo/FreeRTOS-IoT-Libraries-LTS-Beta2/mqtt/mqtt_multitask/DemoTasks/MultitaskMQTTExample.c#L58

Reply:
Please delete when you are porting the code to amazon freertos and notice unnecessary things.